README.md

```diff
@@ -1,24 +1,146 @@
-# README
+# Budget App Rails
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [Budget App](#budget-app)
+- [Budget App ](#budget-app-)
+  - [video demo 🎥](#video-demo-)
+  - [Render Deploy 🚀](#render-deploy-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License ](#-license-)
 
-Things you may want to cover:
+* Configuration
 
-* Ruby version
+# Budget App <a name="about-project"></a>
 
-* System dependencies
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
 
-* Configuration
+## video demo 🎥
+
+[Video demo here](https://www.loom.com/share/edb4f04d81e64e20b2953e79e519982d)
```

Comment:
- [ ] As the previous reviewer required, kindly record a new presenting video and don't forget to talk about a highlight about a piece of the code you find interesting or are proud of making. 🙏

Reply:
@tugsem Thanks for your time and effort. I have taken into consideration your remarks and here is the link for the newly recorded video https://www.loom.com/share/1b2427fb867c4e1c8d2419d57991c1f4.